data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1660 @@
+			`The Remnant have a shocked expression, and a couple of them are looking dazed. Most, however, seem to be handling the incursion fairly well. Dusk gestures eagerly to you. "Good to see you recovered quickly, <first>. We will take care of everyone here - we need you out there. Go do a sweep through each of the surrounding systems, and see if you can get a signal to any of the Remnant worlds. Our communicators have all lost connection, as have the isolated backups. So we need to re-establish communication promptly." He shakes his head. "I love meeting new beings, but they always manage to surprise."`
+				accept
+	on waypoint
+		dialog
+			`You have successfully visited all the nearby systems. Time to report back to <origin>.`
+	on complete
+		conversation
+			`Back in the control room the Remnant appear to have regained their composure. Remnant move quickly from station to station, and every sensor and tool is in use. It is as if the Remnant are afraid everything is going to vanish any moment. Which, you realize, might actually happen. You find Gavriil carefully tending a console, and he looks up as you approach. "The Embers guided you back, I see. How does the outside world fare?" He listens intently as you described your flight through the surrounding systems, and nods when you mention that your navigation system is reporting no connections to the rest of the Ember Waste. Even predictions as to jump drive performance indicate no valid jumps. "No connections whatsoever?" He looks thoughtful. "That actually makes sense." He gestures at a display of the area. "As best our sensors can pick up, the universe beyond this pocket of the Ember Waste just does not exist."`
+			`	He quickly continues. "Of course, we can see it out there. But it is like looking through a very thin film." He looks thoughtful. "The current leading theory is that we are in some kind of dimensional bubble. A pocket dimension of sorts. But we have not yet reached a consensus."`
+			choice
+				`	"What happened to us? Did we all see the same thing?"`
+				`	"Where did those feelings and images come from?"`
+			`	Gavriil frowns in thought. "Based on our initial survey, each person had a slightly different experience. We think that everyone had basically the same event, but as it was a series of impressions, images, and feelings, each person picked up on different aspects of it."`
+			`	He gestures take on a confident tone. "Which makes sense, since our life experiences shape our perceptions. Still, as to what it was..." He pauses, gathering his thoughts. "Our best guess is that this was first contact with some sort of entity." He grins at that. "I know, stating the obvious. But anyway, Dusk thinks that it might be some sort of solitary entity, as it did not use words or meanings, despite having access to our minds. As such, it uses internals to convey meaning instead of words." He glances at a map of the system. "There is, however, no indication as to where it came from.`
```

Comment:
```suggestion
			`Back in the control room the Remnant appear to have regained their composure. Remnant move quickly from station to station, and every sensor and tool is in use. It is as if the Remnant are afraid everything is going to vanish any moment. Which, you realize, might actually happen. You find Gavriil carefully tending a console, and they look up as you approach. "The Embers guided you back, I see. How does the outside world fare?" He listens intently as you described your flight through the surrounding systems, and nods when you mention that your navigation system is reporting no connections to the rest of the Ember Waste. Even predictions as to jump drive performance indicate no valid jumps. "No connections whatsoever?" They look thoughtful. "That actually makes sense." They gesture at a display of the area. "As best our sensors can pick up, the universe beyond this pocket of the Ember Waste just does not exist."`
			`	They quickly continue. "Of course, we can see it out there. But it is like looking through a very thin film." He looks thoughtful. "The current leading theory is that we are in some kind of dimensional bubble. A pocket dimension of sorts. But we have not yet reached a consensus."`
			choice
				`	"What happened to us? Did we all see the same thing?"`
				`	"Where did those feelings and images come from?"`
			`	Gavriil frowns in thought. "Based on our initial survey, each person had a slightly different experience. We think that everyone had basically the same event, but as it was a series of impressions, images, and feelings, each person picked up on different aspects of it."`
			`	Their gestures take on a confident tone. "Which makes sense, since our life experiences shape our perceptions. Still, as to what it was..." They pause, gathering their thoughts. "Our best guess is that this was first contact with some sort of entity." They grin at that. "I know, stating the obvious. But anyway, Dusk thinks that it might be some sort of solitary entity, as it did not use words or meanings, despite having access to our minds. As such, it uses internals to convey meaning instead of words." They glance at a map of the system. "There is, however, no indication as to where it came from.`
```